packages/costa/benchmark/makefile

```diff
@@ -0,0 +1,4 @@
+export DEBUG=costa*
```

Comment:
Why not use the npm script but make?

Reply:
Makefile is more simple for this use-case, and this won't be running at userland.